frontend/CropGuide.css

```
@@ -545,13 +545,22 @@
-  align-items: center;
+  align-items: flex-start;
   gap: 16px;
   margin-bottom: 20px;
+  flex-wrap: wrap;
 }
 
 .modal-header-row h2 {
   margin: 0;
+  flex: 1;
+  min-width: 0;
+  word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)